### PR TITLE
Created associations between classes

### PR DESCRIPTION
**Pull request by erickhansx**
## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] Created associations between student and person. 
- [ ] Created associations between book and rentals 
- [ ] Tested functionality

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Hans, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure that your ReadMe for this project complies with the following standard(s):
- [x]  The _Future Features_ section of your ReadMe includes 1-3 potential features to be added later. 
- [x] Kindly ensure that provided text instructions are removed and filled in with custom content applicable to your project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by rloterh** [comment]
Hi Hans, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure that your ReadMe has the _Future Features_ section which should include 1-3 potential features to be added later. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: classroom.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class Classroom
+  attr_accessor :label, :students
```
Comment (rloterh) [inline classroom.rb]: - [ ] Following best practices, we only intend to _access (get)_ `students` object and therefore it should be marked with `attr_reader`. Marking it with `attr_accessor` may expose our object to unwanted modifications. Kindly ensure to improve upon this aspect.

File: student.rb [hunk 2]
```diff
@@ -1,12 +1,19 @@
 require './person'
```
Comment (rloterh) [inline student.rb:1]: - [ ] OPTIONAL
By convention, it's a good practice to use `require_relative` classes in the same source and use `require` for external modules or packages. Kindly ensure to improve upon this aspect.

File: README.md [hunk 3]
```diff
@@ -48,6 +43,16 @@
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+### Key Features <a name="#future-features"></a>
```
Comment (rloterh) [inline README.md:51]: - [ ] OPTIONAL
Kindly modify this section title from _Key Features_ to _Future Features_ accordingly.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Hans, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project, but you are almost there! 🚦 🚦   🏃   

## Highlights 🎯   
- Nice job so far with creating associations among objects!

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions 🧰 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

_Kindly check the inline comments marked **[OPTIONAL]** under the review._

**You can also consider:**
- MIT license link in your ReadMe is broken, kindly endeavor to fix it, and remove the guide note below it.

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AnselemOdims — APPROVED** [review]
### Hi  @erickhansx 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions
- [x] Kindly implement the optional suggestions made by the previous reviewer.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**